src/components/postView/view/postDisplayView.js

```diff
@@ -221,7 +221,6 @@ const PostDisplayView = ({
                 name={author || post.author}
                 currentAccountUsername={name}
                 reputation={post.author_reputation}
-                content={post}
```

Comment:
@noumantahir Are you sure we don't need this props anymore?

Reply:
@feruzm 
![Screenshot 2021-07-24 at 8 59 48 PM](https://user-images.githubusercontent.com/6298342/126896959-12e31619-67b0-4937-a48a-ba0696af7c26.png)

Reply:
ok